src/models/restful-model-collection.js

```diff
@@ -181,16 +181,18 @@ export default class RestfulModelCollection {
       return Promise.reject(err);
     }
 
+    if (isFunction(body)) {
+      callback = body;
+      body = {};
+    }
+
     const id = itemOrId.id ? itemOrId.id : itemOrId;
+    const deleteBody = itemOrId.id ? itemOrId.deleteRequestBody() : body;
 
-    if (isFunction(params)) {
-      callback = params;
-      params = {};
-    }
```

Comment:
Wat?

Reply:
[discussed in person but commenting for posterity] optional positional args.